tools/index.d.ts

```diff
@@ -0,0 +1,154 @@
+    keep_infinity?: boolean;
+    loops?: boolean;
+    negate_iife?: boolean;
+    passes?: number;
+    properties?: boolean;
+    pure_funcs?: string[];
+    pure_getters?: boolean | 'strict';
+    reduce_funcs?: boolean;
+    reduce_vars?: boolean;
+    sequences?: boolean;
+    side_effects?: boolean;
+    switches?: boolean;
+    toplevel?: boolean;
+    top_retain?: boolean;
+    typeofs?: boolean;
+    unsafe?: boolean;
+    unsafe_arrows?: boolean;
+    unsafe_comps?: boolean;
+    unsafe_Function?: boolean;
+    unsafe_math?: boolean;
+    unsafe_methods?: boolean;
+    unsafe_proto?: boolean;
+    unsafe_regexp?: boolean;
+    unsafe_undefined?: boolean;
+    unused?: boolean;
+    warnings?: boolean;
+}
+
+export enum InlineFunctions {
+    Disabled = 0,
+    SimpleFunctions = 1,
+    WithArguments = 2,
+    WithArgumentsAndVariables = 3
+}
+export interface MangleOptions {
+    eval?: boolean;
+    keep_classnames?: boolean;
+    keep_fnames?: boolean;
+    module?: boolean;
+    reserved?: string[];
+    toplevel?: boolean;
+    safari10?: boolean;
+    properties?: boolean | ManglePropertiesOptions;
+}
+
+export interface ManglePropertiesOptions {
+    builtins?: boolean;
+    debug?: boolean;
+    keep_quoted?: boolean;
+    regex?: RegExp;
+    reserved?: string[];
+}
+
+export interface OutputOptions {
+    ascii_only?: boolean;
+    beautify?: boolean;
+    braces?: boolean;
+    comments?: boolean | 'all' | 'some' | RegExp;
+    ecma?: ECMA;
+    indent_level?: number;
+    indent_start?: boolean;
+    inline_script?: boolean;
+    is8?: boolean;
+    keep_quoted_props?: boolean;
+    max_line_len?: boolean;
+    preamble?: string;
+    preserve_line?: boolean;
+    safari10?: boolean;
+    quote_keys?: boolean;
+    shorthand?: boolean;
+    source_map?: SourceMapOptions;
+    quote_style?: OutputQuoteStyle;
+    semicolons?: boolean;
+    shebang?: boolean;
+    webkit?: boolean;
+    width?: number;
+    wrap_iife?: boolean;
+}
+
+export enum OutputQuoteStyle {
+    PreferDouble = 0,
+    AlwaysSingle = 1,
+    AlwaysDouble = 2,
+    AlwaysOriginal = 3
+}
+
+export interface MinifyOptions {
+    warnings?: boolean | 'verbose';
+    parse?: ParseOptions;
+    compress?: boolean | CompressOptions;
+    mangle?: boolean | MangleOptions;
+    output?: OutputOptions;
+    sourceMap?: boolean | SourceMapOptions;
+    toplevel?: boolean;
+    nameCache?: object;
+    ie8?: boolean;
+    keep_fnames?: boolean;
+    ecma?: ECMA;
+    keep_classnames?: boolean;
+    module?: boolean;
+    safari10?: boolean;
+}
+
+export interface MinifyOutput {
+    error?: Error;
+    warnings?: string[];
+    code: string;
+    map: string;
+}
+
+export interface SourceMapOptions {
+    includeSources?: boolean;
+    filename?: string;
+    url?: string | 'inline';
+    root?: string;
+    content?: RawSourceMap;
+}
+
+export function minify(files: string | string[] | { [file: string]: string }, options?: MinifyOptions): MinifyOutput;
```

Comment:
There are other Terser exports:

https://github.com/terser-js/terser/blob/f5e3a6218ac5345b21d1c7bb02eba342ab4874c7/tools/exports.js#L1-L6

Reply:
ok, i will add it soon.